tests/potential/test_potential.py

```diff
@@ -60,38 +60,33 @@ def test_accessors(self):
         self.assertEqual(m['B','B']['energy'], None)
         self.assertEqual(m['B','B']['mass'], None)
 
-        m['A','B']['energy'] = 1.0
```

Comment:
There is a lot going on in these tests, so let's split them into two methods (two `test_` methods). First, do a test with just one type 'A', and check all these different scenarios. Then, you can do a test with multiple types for a limited scenario or two.